test/test_cli.py

```diff
@@ -0,0 +1,36 @@
+import os
+import pytest
+
+import tempfile
+
+import shutil
+
+from wof.wofpy_config import makedirs
+
+
+@pytest.fixture
+def mkdtemp():
+    _directory = tempfile.mkdtemp()
+    _stat = os.stat(_directory)
+    yield _directory, _stat
```

Comment:
@lsetiawan this is a nice `pytest` trick. This generator, as a `pytest.fixture`, will create the test directory that will be removed only after the test is finished. It can be used with variables, files, etc, anything that you want to clean up after the use. Much less boilerplate than Python's `unittest` module.